src/PIL/Image.py

```diff
@@ -2633,10 +2633,16 @@ def seek(self, frame: int) -> None:
         """
 
         # overridden by file handlers
-        if frame != 0:
```

Comment:
Does it make impossible to call super().seek() for plugins prior Pillow 11?

Reply:
...yes, an external plugin would fail calling `super().seek()` if the user had Pillow < 11.

I could add a dedicated new method instead of using `super().seek()`, but any way you look at the idea of this PR, it is new functionality.